ConferenceV2/app/services/singletons/conference_call_manager.py

```
@@ -90,12 +113,13 @@ async def start_conference_call(self, conf_id: str) -> None:
                 logger_instance.error("Failed to persist CONFERENCE_START_FAILED", exc_info=True)
             raise
-        
+
     def get_conference(self, conference_id: str) -> ConferenceCall | None:
         return self.conferences.get(conference_id, None)
 
-    def delete_conference(self, conf_id: str):
-        del self.conferences[conf_id]
+    def delete_conference(self, conf_id: str) -> None:
+        self.conferences.pop(conf_id, None)
+        asyncio.create_task(self._get_redis().delete(conf_id))
```

Ruff (0.15.15)

[warning] Store a reference to the return value of `asyncio.create_task`

(RUF006)